include/exadg/structure/time_integration/driver_steady_problems.cpp

```diff
@@ -124,8 +124,8 @@ DriverSteady<dim, Number>::do_solve()
     // calculate right-hand side vector
     pde_operator->compute_rhs_linear(rhs_vector, 0.0 /* time */);
 
-    unsigned int const N_iter_linear =
-      pde_operator->solve_linear(solution, rhs_vector, 0.0 /* no mass term */, 0.0 /* time */);
+    unsigned int const N_iter_linear = pde_operator->solve_linear(
+      solution, rhs_vector, 0.0 /* no mass term */, 0.0 /* time */, false);
```

Comment:
Could you write `false /* update of preconditioner */` to be line with the previous two arguments? In case we do not pass variables with a concrete name but "numbers", it helps to understand the code or interpret the parameters passed to a function (no need to open the declaration of that function).

Reply:
I added the comment!